models/organization/team.go

```diff
@@ -249,7 +249,7 @@ func (t *Team) UnitEnabled(tp unit.Type) bool {
 
 // UnitAccessMode returns if the team has the given unit type enabled
 // it is called in templates, should not be replaced by `UnitAccessModeCtx(ctx ...)`
-func (t *Team) UnitAccessMode(tp unit.Type) perm.AccessMode {
+func (t *Team) UnitAccessMode(tp unit.Type) perm.AccessMode { // Notice: It will be used in template, don't remove it directly
```

Comment:
In theory we have `Context` in templates.

Reply:
Yes, but I will leave it for other PRs, this PR is big enough